package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "http-signature",
   "description": "Reference implementation of Joyent's HTTP Signature scheme.",
-  "version": "1.3.6",
+  "version": "1.4.0",
   "license": "MIT",
-  "author": "Joyent, Inc",
+  "author": "MNX Cloud (mnx.io)",
   "contributors": [
     "Mark Cavage <mcavage@gmail.com>",
```

Comment:
Add yourself here?

Reply:
I'd prefer to omit myself from that list since it's such a small change.